src/plenigo/models/AppAccessData.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace plenigo\models;
+
+/**
+ * Description of AppAccessData
+ *
+ * @author Sebastian Dieguez <s.dieguez@plenigo.com>
+ */
+class AppAccessData {
+
+    /**
+     * The given Customer ID
+     *
+     * @var string
+     */
+    private $customerId;
+
+    /**
+     * The customer App ID
+     * 
+     * @var string 
+     */
+    private $customerAppId;
+
+    /**
+     * The Product ID for this App
+     *
+     * @var string
+     */
+    private $productId;
+
+    /**
+     * The App ID description
+     * 
+     * @var string
+     */
+    private $description;
+
+    /**
+     * Constructor with parameters
+     * 
+     * @param string $customerId The Customer ID provided in the request
+     * @param string $customerAppId The customer App ID
+     * @param string $productId The Product ID for this App
+     * @param string $description The App ID description
+     */
+    public function __construct($customerId, $customerAppId, $productId, $description) {
+        $this->customerId = $customerId;
+        $this->customerAppId = $customerAppId;
+        $this->productId = $productId;
+        $this->description = $description;
+    }
+
+    /**
+     * Maps the respons of the GetAllApps call to a list of single access objects
+     * 
+     * @param array $map the array of maps representing the objects
+     * @return array an array of AppAccessData objects
+     */
+    public static function createFromMapArray(array $map = array()) {
+        $res = array();
+        foreach ($map['apps'] as $aData) {
+            array_push($res, AppAccessData::createFromMap($aData));
+        }
+
+        return $res;
+    }
+
+    /**
+     * Maps an associative array to the AppAccessData object
+     * 
+     * @param array $map an associative array representing the AppAccessData object
+     * @return \plenigo\models\AppAccessData
```

Comment:
Cant customize that in BetBeans
http://stackoverflow.com/questions/5922828/how-do-i-customize-netbeans-php-auto-docblock-for-methods-and-attributes

Should I switch to Eclipse? Because the formatter cant enforce that either...